README.md

```diff
@@ -1,14 +1,67 @@
-# Cosmos
+<p align="center">
 
-[![Build status](https://ci.appveyor.com/api/projects/status/kust7g5dlnykhkaf/branch/master?svg=true)](https://ci.appveyor.com/project/CosmosOS/cosmos/branch/master)
-[![Join the chat at (https://discord.com/invite/kwtBwv6jhD](https://img.shields.io/discord/833970409337913344)](https://discord.com/invite/kwtBwv6jhD)
+  <img src="https://cosmosos.github.io/images/logo.png" width="25%" />
 
-Cosmos is an operating system "construction kit" made for Visual Studio 2022. Build your own OS using managed languages such as C#, VB.NET, and more!
+</p>
 
-For instructions on how to install and use Cosmos, please visit the [Cosmos website](http://www.gocosmos.org).
+<h1 align="center">Cosmos</h1>
+
+<p align="center">
+
+  <a href="https://ci.appveyor.com/api/projects/status/kust7g5dlnykhkaf/branch/master">
+    <img src="https://ci.appveyor.com/api/projects/status/kust7g5dlnykhkaf/branch/master?svg=true" />
+  </a>
+
+  <img src="https://img.shields.io/github/languages/code-size/CosmosOS/Cosmos" />
+  <img src="https://img.shields.io/github/downloads/CosmosOS/Cosmos/total" />
+
+  <a href="https://github.com/CosmosOS/Cosmos/releases/latest">
+    <img src="https://img.shields.io/github/v/release/CosmosOS/Cosmos" />
+  </a>
+
+  <a href="https://github.com/CosmosOS/Cosmos/blob/master/LICENSE.txt">
+    <img src="https://img.shields.io/github/license/CosmosOS/Cosmos" />
+  </a>
+
+  <a href="https://github.com/CosmosOS/Cosmos/issues">
+    <img src="https://img.shields.io/github/issues/CosmosOS/Cosmos" />
+  </a>
+
+  <a href="https://discord.com/invite/kwtBwv6jhD">
+    <img src="https://img.shields.io/discord/833970409337913344" />
+  </a>
+
+</p>
+
+<hr/>
 
+Cosmos (C# Open Source Managed Operating System) is an operating system development kit which uses dotnet, along with the custom IL2CPU compiler to convert (most) C# code into a working bare-metal OS.
```

Comment:
`dotnet` -> `.NET`, as we use the default .NET runtime library as a base
`along with the custom IL2CPU [...]` -> `alongside the custom IL2CPU [...]`